createdisk.sh

```diff
@@ -123,7 +129,7 @@ download_podman $podman_version
 get_dest_dir_suffix "${OPENSHIFT_VERSION}"
 destDirSuffix="${DEST_DIR_SUFFIX}"
 
-libvirtDestDir="crc_libvirt_${destDirSuffix}"
+libvirtDestDir="${LIBVIRT_BUNDLE_PREFIX}_${destDirSuffix}"
 mkdir "$libvirtDestDir"
```

Comment:
Could also be 
```
bundleType=""
if [[ ${OKD_VERSION} != "none" ]]
    bundleType="_okd"
fi

libvirtDestDir="crc${bundleType}_libvirt_${destDirSuffix}"
```

This only needs one variable instead of 3. Or could be `destDirPrefix="crc_okd"` if that is more readable/understandable